### ISSUE TITLE
Build Eclipse OpenJ9 on Windows

### ISSUE THREAD

**Issue by mleipe** [ibmruntimes/openj9-openjdk-jdk9#18]
It is desirable that Eclipse OpenJ9 build and run on Windows.

@vsebe is working on this.

Note that I've created a [sibling issue](https://github.com/eclipse/openj9/issues/55) in https://github.com/eclipse/openj9/ as some of the work may need to be done there.

**Comment by vsebe** [comment]
Depends on issues #31 and #32

**Comment by vsebe** [comment]
I had to hack the environment to use the right linker, the msvs not the version packed by cygwin and ensure there is one `link` on the path:

```
$ which link
/cygdrive/c/Program Files (x86)/Microsoft Visual Studio 12.0/VC/bin/link

```
This hack and all of the above changes are sufficient to build it and run java -version.

**Comment by gdams** [comment]
@vsebe any idea why this is failing for me? https://ci.adoptopenjdk.net/job/openjdk9_openj9_build_x86-64_windows/3/console

**Comment by gdams** [comment]
also I wouldn't recommend building using one of our nighties as a boot JDK (https://adoptopenjdk.net/releases.html?variant=openjdk8#x64_win) would be a better binary to use

**Comment by vsebe** [comment]
The change merged in PR #66 breaks the build jdk on windows.
 @gdams please wait until I fix it.

**Comment by vsebe** [comment]
Fixed.

**Comment by groeges** [comment]
@vsebe Is this still being worked on. If it is, what is still outstanding to get this working?

**Comment by cretz** [comment]
@vsebe, close without comment? Can you at least link to the issue tracking availability of prebuilt binaries on adoptopenjdk.com? Is it https://github.com/eclipse/openj9/issues/55?

**Comment by vsebe** [comment]
@gdams any progress on https://ci.adoptopenjdk.net/job/openjdk9_openj9_build_x86-64_windows/?

**Comment by gdams** [comment]
We are currently waiting on getting a build machine with VS2013. Our only build machine currently does everything on VS2010. Once we have that setup we should be able to start building

**Comment by pshipton** [comment]
Also note the performance of the Windows build won't be optimal because of https://github.com/eclipse/openj9/issues/727

**Comment by pdbain-ibm** [comment]
Please add instructions for installing mingw, which is required in order to fix issue #727.

**Comment by pshipton** [comment]
@pdbain-ibm please provide those instructions, once you have it working. It would be a good idea to supply those instructions in a new issue, created on OpenJ9, and not here in this closed issue on the wrong repo.
